app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +31,30 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            setButonLable(text);
+
+            if (text == "something") {
+                setButonLable("something");
+            } else           if (text == "something-else") {
+                setButonLable("something-else");
+            }else           if (text == "something-else") {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/fe7e7528f1f744e604f2b7cc9ce59e926f994e92/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L43"></span>WhitespaceAround: ‘else’ is not preceded with whitespace.</td>
    </tr>
  </tbody>
</table>